### ISSUE TITLE
Expose API about the VS Code version of the currently running instance.

### ISSUE THREAD

**Issue by hsinyinfu** [microsoft/vscode#139626]
Is it possible to expose an API so that extensions can query the version of current VS Code instance running?
It would be better if the API can also provide the information about installation mode or portable mode.

In our company, we plan to deploy VS Code as main IDE for our employee,
and it is great if VS Code API could provide version information for extension developers to query, 
so that we can develop our extensions based on the VS Code version information.

**Comment by jrieken** [comment]
There is `require('vscode').version` for that. Did you miss that or isn't it suffient?

**Comment by jrieken** [comment]
https://github.com/microsoft/vscode/blob/f33cb00d873583d24210beb526f6489508410146/src/vscode-dts/vscode.d.ts#L11

**Comment by hsinyinfu** [comment]
Hi, thanks for the quick response.
I'm sorry that I didn't aware this information has been provided in the vscode package we imported.
I just tried and it works find.

I would like to know that is this information varies when user open multiple version of VS Code instances?
For example, if we develop an extension to show the version information retrieved from "vscode.version", and our user open a 1.61.2 and a 1.62.1 VS Code windows with this extension installed, it would show the corresponding version on both VS Code instances (show 1.61.2 on 1.61.2 VS Code instance, and show 1.62.1 on 1.62.1 VS Code instance), right?
